### PR TITLE
feat: add options menu

### PR DESCRIPTION
**Pull request by pkulev**
For now player can set music and SFX volume using sliders. Maybe later with persisted
game settings CLI arguments can pass away.

Signed-off-by: Pavel Kulyov <kulyov.pavel@gmail.com>

### CODE REVIEW

**Comment by moonburnt** [comment]
That's awesome, but there is a bug: when exiting options without saving, you dont reset slider's value to its initial state.

Also, if possible, it could be nice if you used base.music_player.set_player_volume() instead of base.musicManager.set_volume() - both get the job done, but since music_player is already wrapper on top of musicManager, its usage is encouraged

File: Game/interface.py [hunk 1]
```diff
@@ -168,6 +182,101 @@ def __init__(self, play_command, exit_command):
...
+    def save_and_close(self):
+        '''Save options and return to parent menu.'''
+
+        log.info("Saving options...")
+        shared.MUSIC_VOLUME = self.music_volume
+        shared.SFX_VOLUME = self.sfx_volume
+
+        self.back_command()
+
+    def restore_and_close(self):
```
Comment (moonburnt) [inline Game/interface.py:265]: This should also reset state of sliders to defaults

File: Game/interface.py [hunk 2]
```diff
@@ -168,6 +182,101 @@ def __init__(self, play_command, exit_command):
...
+        '''Restore previous options' values and return to parent menu.'''
+
+        base.musicManager.set_volume(self.old_music_volume)
```
Comment (moonburnt) [inline Game/interface.py]: Use base.music_player.set_player_volume() instead (here and in other cases)

**Review by moonburnt — CHANGES_REQUESTED** [review]

File: Game/interface.py (hunk 1 above)
Comment (pkulev) [reply Game/interface.py:265]: Done

File: Game/interface.py (hunk 2 above)
Comment (pkulev) [reply Game/interface.py]: Done

**Comment by moonburnt** [comment]
Awesome, thanks so much

**Pull request merged**